communication/socket/create-vmci-socket.yml

```
@@ -5,25 +5,27 @@ rule:
     authors:
       - jakub.jozwiak@mandiant.com
     scopes:
-      static: function
+      static: basic block
       dynamic: thread
     mbc:
       - Communication::Socket Communication::Create Socket [C0001.003]
     references:
       - https://www.vmware.com/products/beta/ws/VMCIsockets.pdf
     examples:
-      - 9ed5660c6a442dbba9e2ba795ccc913c1f1517ce89854fe4287c1c8b36b21d52:0x180001241
+      - 9ed5660c6a442dbba9e2ba795ccc913c1f1517ce89854fe4287c1c8b36b21d52:0x1800011D0
   features:
     - or:
       - and:
         - os: windows
-        - api: DeviceIoControl
-        - number: 0x81032068 = VMCI_SOCKETS_GET_AF_VALUE
-        - optional:
+        - or:
           - api: socket
+          - api: DeviceIoControl
+        - number: 0x81032068 = VMCI_SOCKETS_GET_AF_VALUE
       - and:
         - os: linux
-        - api: ioctl
-        - number: 0x7B8 = VMCI_SOCKETS_GET_AF_VALUE
+        - call:
+          - and:
+            - api: ioctl
+            - number: 0x7B8 = VMCI_SOCKETS_GET_AF_VALUE
         - optional:
           - api: socket
```

Not sure why there's an added `call` subscope here. Can't we omit that? Also, can't we create VMCI sockets using `socket()` on linux and therefore do:
```yaml
- and:
  - or:
    - api: socket
    - api: ioctl
  - number: 0x7B8 = VMCI_SOCKETS_GET_AF_VALUE
```

```suggestion
        - or:
          - api: ioctl
          - api: socket
        - number: 0x7B8 = VMCI_SOCKETS_GET_AF_VALUE
```

---

no, socket does not support this argument, it's just a related (optional) call